mixer/pkg/attribute/envoyProtoBag_test.go

```diff
@@ -0,0 +1,198 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package attribute
+
+import (
+	"net"
+	"testing"
+	"time"
+
+	core "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
+	authzGRPC "github.com/envoyproxy/go-control-plane/envoy/service/auth/v2"
+	"github.com/golang/protobuf/ptypes/timestamp"
+
+	attr "istio.io/pkg/attribute"
+	"istio.io/pkg/log"
+)
+
+func TestBagEnvoy(t *testing.T) {
+	attrs := &authzGRPC.CheckRequest{
```

Comment:
could you use the helper method (`envoyProtoAttrsForTesting`) for gen'ing attrs here too?  maybe make everything more readable?